woocommerce/woocommerce
# [3.6] Only track product page marketplace suggestions when visible.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes the tracks event for marketplace suggestions on the edit product page.

### How to test the changes in this Pull Request:

1. Edit a Product
2. Verify that tracks events are not fired for suggestions on page load
3. Click "get more options"
4. Verify that tracks events are fired for suggestions
5. Click "get more options" again
6. Verify that tracks events are not fired for suggestions
